### ISSUE TITLE
evaluate issue

### ISSUE THREAD

**Issue by jxmlxt** [graphdeco-inria/gaussian-splatting#1042]
I use the same set of colmap data as input and train with the same number of iterations. Why do the final indicators such as SSIM and PSNR have significant fluctuations。Thanks！

**Comment by jxmlxt** [comment]
![image](https://github.com/user-attachments/assets/424dd619-5b00-4193-b8ab-4c52ddd6cdaa)
first

**Comment by jxmlxt** [comment]
![image](https://github.com/user-attachments/assets/0d9ddf89-fab1-4964-b466-b427fdb7ef77)
second

**Comment by alanvinx** [comment]
Hi, I see that you evaluate using only 4 renders, did you look at them? With this few test images if a floater apears in one of the test views in the first model but not in the second model then PSNR will be a lot lower.

**Comment by jxmlxt** [comment]
Because I am using small scene input with only 26 images in total, why does this situation occur? Can't small scenes be qualitatively evaluated? I checked the first set of images and did not find any floating objects. Thank you for your reply

**Comment by jxmlxt** [comment]
Also, is the fluctuation of SSIM normal? Why is there such a large fluctuation.Thanks!

**Comment by AsherJingkongChen** [comment]
It may be some random generators (on training pipeline) don't have a fixed seed.

I see you screenshot two different models.
You can try evaluate on exactly the same `point_cloud.ply`, where you can ensure the inputs don't differ.

**Comment by jxmlxt** [comment]
thanks for your reply!You're right!While I have fixed the random seed in the train.py.the situation is still exist!I will check the pipe again.

**Comment by AsherJingkongChen** [comment]
@jxmlxt I guess you find that two point cloud files are different? I think there is a `seed` option missing in [`densify_and_split`](https://github.com/graphdeco-inria/gaussian-splatting/blob/54c035f7834b564019656c3e3fcc3646292f727d/scene/gaussian_model.py#L420).

**Comment by jxmlxt** [comment]
I have a try，while it doesn't work.I don't understand how they test.Just use the best score?

**Comment by jxmlxt** [comment]
@AsherJingkongChen You're right.I have a try，while it doesn't work.I don't understand how they test.Just use the best score?

**Comment by AsherJingkongChen** [comment]
> @AsherJingkongChen You're right.I have a try，while it doesn't work.I don't understand how they test.Just use the best score?

It should be called "experiments", which does not guarantee reproducibility.

**Comment by Yang-Xijie** [comment]
@jxmlxt Have you figured out which part of the code introduces such fluctuation?
related issue: https://github.com/graphdeco-inria/gaussian-splatting/issues/89
